tests/tck/features/vector/AnnSearch.feature

```diff
@@ -0,0 +1,190 @@
+Feature: Ann Search Tests
+
+  Background:
+    Given an empty graph
+    And create a space with following options:
+      | partition_num  | 1                |
+      | replica_factor | 1                |
+      | vid_type       | FIXED_STRING(30) |
+      | charset        | utf8             |
+      | collate        | utf8_bin         |
+
+  Scenario: Test Ann Search
+    # Create tags with vector properties
+    When executing query:
+      """
+      CREATE TAG player_vector(name string, age int, embedding vector(3));
+      CREATE TAG coach_vector(name string, experience int, embedding vector(3));
+      """
+    Then the execution should be successful
+    And wait 6 seconds
+    # Insert test data - players
+    When executing query:
+      """
+      INSERT VERTEX
+        player_vector(name, age, embedding)
+      VALUES
```

Comment:
better to design a test dataset with fixed data pattern to facilitate result verification.